Dockerfile.ubuntu

```diff
@@ -43,11 +44,18 @@ RUN apt-get install -y cmake libboost-python-dev python-dev
 # ZBar
 RUN apt-get install -y libzbar-dev python-dev
 
+# pip (need a recent version e.g. for wheels support - ubuntu is obsolete)
+RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN python get-pip.py
```

Comment:
```
# Pip - install latest version so that wheels are used
RUN apt-get install -y python-pip && \
  pip install -U pip
```
works, but gives:
```
Installing collected packages: pip
  Found existing installation: pip 1.5.4
    Not uninstalling pip at /usr/lib/python2.7/dist-packages, owned by OS
Successfully installed pip
```
which gives me the impression that now 2 different pips are installed. I don't like that.

What is reasoning behind your comment? You would like to avoid the `wget` of a mysterious file that gets installed into the system then? Security issues?